### DISCUSSION TITLE
Creating a pre-release using npm publish tag doesn't seems to work, always set as latest

### DISCUSSION THREAD

**Discussion by esakal** [community/community#59884] [Packages]
### Select Topic Area

Bug

### Body

Hello,
I'm trying to have distribution channels in Github Packages. I have a `latest` version as well as `beta`, `rc`, `next`, and some custom ones. But no matter what I do, it always publish it as `latest`

For exampe, I'm running `npm publish --tag next` and I do see in the console log the tag is set to `next`.

```
Publishing to https://npm.pkg.github.com with tag next and default access
```

but it stills deploy the version as `latest` channel and doesn't create a pre-release named `next`. it doesn't matter what the value I provide to tag flag, it always deploy with `latest`

![CleanShot 2023-07-02 at 12 45 29](https://github.com/community/community/assets/4751797/d36a88b8-2662-49b8-b5aa-5ca797c10320)

Please advice
Thanks
Eran

**Comment by esakal** [comment]
Hello,

I wanted to bring your attention to an issue I have been encountering with your library on GitHub. After investing considerable time in troubleshooting, I came across a relevant discussion on GitHub (https://github.com/orgs/community/discussions/54247) that seems to touch upon the same problem I am facing.

It appears that there is a bug within the user interface (UI) of your library's repository, which is proving to be quite elusive to identify. This bug has led to confusion among users, including myself, regarding the behavior of the "latest" version. Although it returns the correct version "v6.0.0", in the UI it indicate that a different not relevant version will be returned "6.0.1-test.2". Also there is no visibility on other tags available to the user.

I kindly request your attention to this matter and urge you to investigate the issue. 

Thank you for your attention.

Best regards,
Eran

**Comment by rushelex** [comment]
The "latest" channel is not only shown in the GitHub Packages interface. The package is actually published in the "latest" channel, even though it says:
> Publishing to https://npm.pkg.github.com with **tag next** and default access

This is completely incorrect behavior for the GitHub Packages registry!

**Comment by jmaixl** [comment]
+1, similar or same issue here as well https://github.com/orgs/community/discussions/54247
